pkg/autoscaler/stats_reporter.go

```diff
@@ -103,6 +115,24 @@ func register() {
 			Aggregation: view.LastValue(),
 			TagKeys:     metrics.CommonRevisionKeys,
 		},
+		&view.View{
+			Description: "Number of pods that are not ready currently",
+			Measure:     notReadyPodCountM,
+			Aggregation: view.LastValue(),
```

Comment:
There's a known issue for `LastValue` metric: https://github.com/knative/serving/issues/4470

I'd @ssmall to put some opinions here.